tasks/build-website-testing.js

```diff
@@ -0,0 +1,8 @@
+var gulp = require('gulp');
+var config = require('./config.js');
```

Comment:
> as this standard, I dont like this, because cause hard diffs in code-review/github, but it is just my opnion

Yeah, it's a common opinion. But from our experience it's not a big deal when it comes to diffs. But on the other hand, variable definitions are more readable.